joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/astcreation/AstCreator.scala

```diff
@@ -1019,6 +1019,9 @@ class AstCreator(
           .code(varSymbol.getText)
           .lineNumber(varSymbol.getLine)
           .columnNumber(varSymbol.getCharPositionInLine)
+        if (arrayParameter != null) {
```

Comment:
Could we perhaps wrap all nullable values in `Option` type? Or implement an implicit class that automatically does this

Reply:
Done. But there are so many places we have to fix it then 😢

Reply:
Lol yeah fair enough